docs/competition/competition-roles.md

```diff
@@ -0,0 +1,55 @@
+# Competition Roles
+
+To run a successful competition, there are a number of roles which need to be assigned. One role may be filled by multiple volunteers, and one volunteer may be filling multiple roles. Some roles should only be assigned to a single volunteer, and this be their only role.
+
+Depending on the nature of the event, some of these roles may not be applicable.
+
+- [Intro Briefer](./event/intro-brief.md)
+- [Refreshments](./volunteers/refreshments.md)
+- [Team Pit Manager](./team-pits/README.md)
+- [Robot Safety Inspector](./matches/README.md#robot-safety-inspections)
+- Group Photo Photographer
+- Health and Safety Coordinator
+## Matches
+
+- [Commentators](./matches/commentating.md)
+- [Judge](./matches/judging.md)
+- [Match Scorer](./matches/match-scoring.md)
+
+### Shepherding
+
+- [Head Shepherd](./matches/shepherding.md#head-shepherd)
+- [Team Shepherds](./matches/shepherding.md#team-shepherds)
+- [Badger](./matches/shepherding.md#badgers)
+- [Roving Shepherd](./matches/shepherding.md#roving-shepherd)
+
+### Arena
+
+- [Marshal](./matches/match-scoring/#match-marshal)
+- [Arena Resetter](./matches/match-scoring/#match-marshal)
+- [Score Entry](./matches/match-scoring/#match-score-entry)
+
+## Team Support
+
+- [Roving Helper](./team-support/roving-helper.md)
+- [Tinker Time Coordinator](./team-support/tinker-time.md)
+- [Livestream Producer](./livestream/README.md)
+
+### Helpdesk
+
+- [Battery Charging](./team-support/battery-charging.md)
+- [Battery Shuffler](./team-support/battery-shuffler.md)
+- [Helpdesker](./team-support/helpdesk.md#helpdesk)
+
+## Reception
+
+- [Photo Studio Manager](./teams/photo-studio.md)
+- [Photo Studio Helper](./teams/photo-studio.md#photo-studio-helper)
+- [Arrival Desk](./teams/role-descriptions/arrival-desk.md)
+- [Crimper](./teams/role-descriptions/crimper.md)
+- [Door Guard](./teams/role-descriptions/door-guard.md)
+- [Kit Return](./teams/role-descriptions/kit-return.md)
+- [Reception](./teams/role-descriptions/reception.md)
+
+!!! note
+  This list may not be complete
```

Comment:
Maybe put this at the top?

Reply:
https://github.com/srobo/runbook/pull/114/commits/2175651fe0ddbeec5af609d4fa8fb004df30eca7